x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -83,6 +85,7 @@ const EndpointListNavLink = memo<{
       className="eui-textTruncate"
       href={href}
       onClick={clickHandler}
+      style={{ color: theme.eui.euiColorInk }}
```

Comment:
think you have a typo here.  Is this supposed to be `euiColorLink` ?

It would explain why this link here doesn't have the right color now.

![image](https://user-images.githubusercontent.com/56395104/112471332-2307be00-8d42-11eb-83c3-18309c00a1f6.png)

Reply:
yup, didn't realize that component was also used in the hostname. was specifically trying to change the font color in the policy status badge because we didn't want it to be blue. i'll make it more specific!